.travis.yml

```diff
@@ -2,16 +2,10 @@ language: generic
 
 env:
   global:
-    - ENCRYPTION_LABEL: "e149beb9c312"
-addons:
-  apt:
-    packages:
-      - oracle-java8-set-default
-install:
-  - curl -O https://sideshowbarker.net/nightlies/jar/vnu.jar
+    - ENCRYPTION_LABEL="e149beb9c312"
+    - DEPLOY_USER="annevankesteren"
 script:
-  - bash ./deploy.sh
-  - /usr/lib/jvm/java-8-oracle/jre/bin/java -jar vnu.jar --skip-non-html /home/travis/build/whatwg/fetch
+  - curl --remote-name --fail https://resources.whatwg.org/build/deploy.sh && bash ./deploy.sh
```

Comment:
I'll leave it for now since that would require instruction changes in whatwg/meta too. Maybe something to ponder once you add support for additional resources to be copied over.